core/server/apps/amp/index.js

```diff
@@ -1,15 +1,26 @@
-var router           = require('./lib/router'),
+var router              = require('./lib/router'),
     registerAmpHelpers  = require('./lib/helpers'),
+    config              = require('../../config'),
+    ampIsEnabled;
 
-    // Dirty requires
-    config     = require('../../config');
+ampIsEnabled = function ampIsEnabled() {
```

Comment:
If i compare this logic to other internal apps, we usually use a middleware for it.
See private blogging app: https://github.com/TryGhost/Ghost/blob/lts/core/server/apps/private-blogging/index.js#L22